code/controllers/subsystem/minimaps.dm

```diff
@@ -46,15 +46,19 @@ SUBSYSTEM_DEF(minimaps)
 				var/turf/location = locate(xval,yval,level)
 				if(isspaceturf(location))
 					continue
-				if(location.density)
+				else if(location.density)
 					icon_gen.DrawBox(location.minimap_color, xval, yval)
```

Comment:
```suggestion
				if(location.density)
					icon_gen.DrawBox(location.minimap_color, xval, yval)
```
theres a continue above this already

Reply:
I just like it for clarity, I'll remove though.